README.md

```
@@ -1,61 +1,409 @@
+
+<summary><h4>📝 License</h4>
+
+Copyright © 2024 [WolfStar][profile-link]. <br /> This project is
+[Apache 2.0](./LICENSE) licensed.
+
+<!-- LINK GROUP -->
+
+[ko-fi-link]: https://ko-fi.com/redstar071
+[patreon-link]: https://www.patreon.com/RedStar071
+[github-sponsor-link]: https://github.com/sponsors/wolfstar-project
+[wolfstar-invite-link]: https://invite.wolfstar.rocks
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "wolfstar-invite-link"

(MD053, link-image-reference-definitions)